source/js/components/join/join.jsx

```diff
@@ -445,11 +450,17 @@ export default class JoinUs extends React.Component {
                 ref={el => (this.privacy = el)}
                 required
               />
-              <p className="d-inline-block body-small form-text mb-0">
-                {getText(
-                  `I'm okay with Mozilla handling my info as explained in this Privacy Notice`
-                )}
-              </p>
+              <p
+                className="d-inline-block body-small form-text mb-0"
+                dangerouslySetInnerHTML={{
```

Comment:
This part looks... questionable. If there's a react-appropriate gettext, it should understand "slotting in more JSX".

Reply:
I must admit I have no idea how to do that…
Of course `<p>{gettext('I’m okay with Mozilla handling my info as explained in this <Link to="{link}">Privacy Notice</Link>')}</p>` won’t be interpreted as JSX